packages/aws-cdk-lib/aws-codebuild/lib/compute-type.ts

```diff
@@ -14,4 +14,5 @@ export enum ComputeType {
   LAMBDA_4GB = 'BUILD_LAMBDA_4GB',
   LAMBDA_8GB = 'BUILD_LAMBDA_8GB',
   LAMBDA_10GB = 'BUILD_LAMBDA_10GB',
+  ATTRIBUTE_BASED_COMPUTE = 'ATTRIBUTE_BASED_COMPUTE',
```

Comment:
`FleetComputeType.ATTRIBUTE_BASED_COMPUTE` feels a little redundant, I went with:

```suggestion
  ATTRIBUTE_BASED = ComputeType.ATTRIBUTE_BASED_COMPUTE,
```